### ISSUE TITLE
make failed, 'I_SWROPT' undeclared

### ISSUE THREAD

**Issue by XiaoYu-HN** [hanslub42/rlwrap#120]
Oracle Linux 7.9, make failed with below error:
```
# make install
Making install in doc
make[1]: Entering directory `/home/opc/rlwrap/doc'
sed -e 's#@DATADIR@#/usr/local/share#'  rlwrap.man > rlwrap.1
make[2]: Entering directory `/home/opc/rlwrap/doc'
make[2]: Nothing to be done for `install-exec-am'.
 /bin/mkdir -p '/usr/local/share/man/man1'
 /bin/install -c -m 644 rlwrap.1 '/usr/local/share/man/man1'
make[2]: Leaving directory `/home/opc/rlwrap/doc'
make[1]: Leaving directory `/home/opc/rlwrap/doc'
Making install in src
make[1]: Entering directory `/home/opc/rlwrap/src'
gcc -DHAVE_CONFIG_H -I. -I..    -DDATADIR=\"/usr/local/share\"  -g -O2 -MT ptytty.o -MD -MP -MF .deps/ptytty.Tpo -c -o ptytty.o ptytty.c
ptytty.c: In function 'ptytty_control_tty':
ptytty.c:258:23: error: 'I_SWROPT' undeclared (first use in this function)
         ioctl(fd_tty, I_SWROPT, 0);
                       ^
ptytty.c:258:23: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [ptytty.o] Error 1
make[1]: Leaving directory `/home/opc/rlwrap/src'
make: *** [install-recursive] Error 1

```

**Comment by hanslub42** [comment]
`rlwrap`s `configure` script assumes that systems that provide `isastream()` provide the whole STREAMS subsystem (including the `I_SWROPT` macro).  This assumption is wrong, as you discovered.

This should be fixed by 143bdc3f17d8cc1ad7d74ada4155ff019630a828. Can you confirm?

**Comment by XiaoYu-HN** [comment]
Thanks a lot, it works after patching!
